server/chart_config.json

```diff
@@ -136,11 +150,25 @@
             "statsVars": [
               "Count_Person_Upto5Years",
               "Count_Person_5To17Years",
+              "Count_Person_15To19Years",
```

Comment:
FYI, Our charts aren't going to look good with so many labels. We'll have to aggregate some of these like we do for GNI.

Reply:
I know this will leave gaps, but can we omit these new ones for now? Ending at 65+ seems reasonable for the long term as well.

Reply:
Done

Reply:
Note US won't use the newly added age group since census does not have these ages. 
Even it is a large group, US and EU only use a fraction of it separately.